components/cstudio-forms/controls/rte.js

```diff
@@ -546,7 +546,7 @@ YAHOO.extend(CStudioForms.Controls.RTE, CStudioForms.CStudioFormField, {
 						ed.contextControl._onChange(null, ed.contextControl);				
 			        });
 			        ed.onChange.add(function(ed, l) {
-			        	ed.contextControl.resizeEditor(ed);
+			        	//ed.contextControl.resizeEditor(ed);
```

Comment:
This looks like it will cause a bug
Also why is it commented out rather than removed?